Superlista/Views/AddCollaboratorSheetView.swift

```diff
@@ -79,7 +79,7 @@ struct AddCollaboratorSheetView: View {
                             if !collaborators.isEmpty {
                                 List {
                                     ForEach(0..<self.collaborators.count, id: \.self) { index in
-                                        CollaboratorListView(collaborators: self.$collaborators, list: list, name: collaborators[index].name!, index: index)
+                                        CollaboratorListView(collaborators: self.$collaborators, list: list, name: collaborators[index].name ?? "", index: index)
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()